### PR TITLE
Added Bequant exchange

### PR DESCRIPTION
**Pull request by peedrr**
As per title. This implements ticker, L2 book, trades, candles, plus authenticated channels: order info, account transactions and account balances.

The two last channels are obviously new, named ACC_TRANSACTIONS AND ACC_BALANCES.

There is a demo_bequant.py which tests all of the new features

- [X] - Tested
- [X] - Changelog updated
- [ ] - Tests run and pass
- [X] - Flake8 run and all errors/warnings resolved
- [ ] - Contributors file updated (optional)

### CODE REVIEW

File: cryptofeed/feed.py [hunk 1]
```diff
@@ -47,6 +47,8 @@ def __init__(self, address: Union[dict, str], timeout=120, timeout_interval=30,
         book_interval: int
             Number of updates between snapshots. Only applicable when book deltas are enabled.
             Book deltas are enabled by subscribing to the book delta callback.
+        candle_interval: str
+            Length of time between a candle's Open and Close.
```
Comment (bmoscon) [inline cryptofeed/feed.py:51]: you might want to mention that this is only a valid kwarg on exchanges that support candles

**Comment by bmoscon** [comment]
@peedrr  it looks like you forgot to add the bequant file in the `exchange` module

**Review by bmoscon — APPROVED** [review]

**Comment by bmoscon** [comment]
looks good - if you dont mind tweaking that comment in feed.py ill merge this

File: cryptofeed/feed.py (hunk 1 above)
Comment (peedrr) [reply cryptofeed/feed.py:51]: Yeah, to be honest, this comment should be at the exchange level but it seems there's no docstring inheritance. Child classes overwrite the parent's docstring, huh?

**Review by bmoscon — APPROVED** [review]

**Pull request merged**